serverless.yml

```diff
@@ -4,7 +4,7 @@ useDotenv: true
 provider:
   region: us-west-2
   name: aws
-  runtime: nodejs14.x
```

Comment:
Node 16 is not yet supported on AWS lambda https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html please revert.

Reply:
That's...surprising.